src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -846,6 +846,16 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="$(RazorDesignTimeTargets)" />
 
+  <PropertyGroup>
+    <!--
+      Defines the list of file extensions that VS will monitor to reload the application.
+      We'll only define these for C# projects targeting RazorLangVersion 3.0 or later, and let VS use defaults in other cases.
+
+      This property can be removed after the next insertion in to VS.
+    -->
+    <UpToDateReloadFileTypes Condition="'$(Language)'=='C#' AND '$(_Targeting30OrNewerRazorLangVersion)' == 'true' AND '$(RazorUpToDateReloadFileTypes)' == ''">$(RazorUpToDateReloadFileTypes)</UpToDateReloadFileTypes>
+  </PropertyGroup>
```

Comment:
Why does this need to be defined twice? the design time targets are included here.

Reply:
Until we do the next insertion, we have a "bad" design time targets that does not know about this property. I'll sync up with @ajaybhargavb, we should be able to remove this line once we do our next insertion in to VS. In the meanwhile this wholesale overrides any value that the design time target has setup.